doc/deprecation_process.md

```diff
@@ -2,57 +2,169 @@ This page can be linked using: [aka.ms/azsdk/python/deprecation-process](https:/
 
-The overall idea is that PyPI do not support an official deprecation logic. We concluded that the best way was:
-- Change the classifier as `Inactive`, to showcase in metadata that this package is longer worked on
-- Add a disclaimer on the main Readme file to explain deprecation, and guide to migration guide to other package as necessary. While a migration guide should always be written, you may decide to postpone this work based on downloads numbers (found on [pypistats](https://pypistats.org/), [pype.tech](https://www.pepy.tech/), etc.) and internal knowledge of the usage of the package.
-- Push a new release to PyPI
+The overall idea is that PyPI does not support an official deprecation logic. We concluded that the best way is:
+- Change the classifier as `Inactive`, to showcase in metadata that this package is longer worked on.
+- Add a disclaimer on the main Readme file to explain deprecation, and provide a migration guide as necessary.
+- Push a new release to PyPI.
+- Update the API reference docs to show the deprecated status of the package.
 
-**Important Note**: The best versioning approach would be to do a [post release](https://peps.python.org/pep-0440/#post-releases). However, due to some tooling issues at the moment, currently version should be the next beta, or the next patch version ([example](https://github.com/Azure/azure-sdk-for-python/commit/cf3bfed65a65fcbb4b5c93db89a221c2959c5bb4)). Follow those issues for details https://github.com/Azure/azure-sdk/issues/7479 and https://github.com/Azure/azure-sdk-tools/issues/5916.
+# Step 1: Updates to the package files
 
-# Step 1: Update in the repository
+Clone the `azure-sdk-for-python` repository and update the following files of your package.
 
-Clone the repository and udpate the following files of your package:
-- `README.MD` add a disclaimer using this syntax:
+## README.md
 
-   > This package is no longer being maintained. Use the [azure-mynewpackage](https://pypi.org/project/azure-mynewpackage/) package instead.
-   >
-   > For migration instructions, see the [migration guide](https://aka.ms/azsdk/python/migrate/my-new-package).
+A disclaimer should be added directing to a replacement package and migration guide as necessary.
+  - The link to the replacement package should be a PyPI link: `https://pypi.org/project/azure-mynewpackage/`.
+  - The link to the migration guide should be an `aka.ms` link in the format `https://aka.ms/azsdk/python/migrate/my-new-package`. To create this link, go to [https://aka.ms/](https://aka.ms/).
+  - **NOTE**: While a migration guide should always be written, you may decide to postpone this work based on downloads numbers (found on [pypistats](https://pypistats.org/), [pype.tech](https://www.pepy.tech/), etc.) and internal knowledge of the usage of the package. 
```

Comment:
```suggestion
  - **NOTE**: While a migration guide should always be written, you may decide to postpone this work based on downloads numbers (found on [pypistats](https://pypistats.org/), [pepy.tech](https://www.pepy.tech/), etc.) and internal knowledge of the usage of the package. 
```

Reply:
Also, would we postpone a migration guide or just not write one if the following conditions were true?

Reply:
Technically, since people may choose to postpone forever, they could never write it. 😄 

But will get clarification and update this to explicitly mention that.

Reply:
updated to say that writing a migration guide may be skipped.